src/main/java/org/folio/rest/impl/UploadBatchVoucherExportHelper.java

```diff
@@ -109,7 +113,10 @@ private Future<Void> updateHolderWithExportConfig(BatchVoucherUploadHolder uploa
           uploadHolder.setExportConfig(exportConfigs.getExportConfigs().get(0));
           return null;
         }
-        throw new HttpException(404, "Batch export configuration was not found");
+        var param = new Parameter().withKey("query").withValue(query);
+        var error = new Error().withMessage("Batch export configuration was not found").withParameters(List.of(param));
```

Comment:
question: is it required to introduce error code or is it okay to skip in this case

Reply:
if we already introduced new error code - better to use it in all places to be consistent